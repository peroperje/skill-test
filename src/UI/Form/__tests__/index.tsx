import React from 'react';
import { create } from 'react-test-renderer';

import { TextField } from '../index';

describe('TextField', () => {
  it('Snapshot', () => {
    const component = create(<TextField />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
