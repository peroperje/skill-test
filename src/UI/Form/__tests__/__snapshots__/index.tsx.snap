// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextField Snapshot 1`] = `
<div
  className="MuiFormControl-root MuiTextField-root"
>
  <div
    className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      className="MuiInputBase-input MuiInput-input"
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
    />
  </div>
</div>
`;
